recipe/meta.yaml

```diff
@@ -1,53 +1,50 @@
 {% set name = "cattrs" %}
-{% set version = "22.1.0" %}
-{% set sha256 = "94b67b64cf92c994f8784c40c082177dc916e0489a73a9a36b24eb18a9db40c6" %}
+{% set version = "23.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/python-attrs/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
why not taking from pypi ?  there is v23.1.2 already, https://pypi.org/project/cattrs/

Reply:
There are no tests in the `pypi` tarball